src/middlewares/enricher.ts

```diff
@@ -0,0 +1,54 @@
+import { Context, Next } from 'koa';
+import {
+  ProcessorTransformationRequest,
+  RouterTransformationRequest,
+  RouterTransformationRequestData,
+} from '../types';
+import GeoLocationHelper from '../helpers/geoLocation';
+
+export type DTRequest = RouterTransformationRequest | ProcessorTransformationRequest[];
```

Comment:
need to export it?

Reply:
Was thinking someone else might want to use it. Hence exported the type.
May be this is not the place for it to reside. Will move this to `types.ts`

Reply:
DTRequest should be expanded

Reply:
Addressed